scripts/env-data.sh

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env bash
-
-DEFAULT_DATADIR="/var/lib/postgresql/12/main"
-ROOT_CONF="/etc/postgresql/12/main"
+POSTGIS_VER=13
```

Comment:
Nice!
I like it.
Maybe we can generalize further by initializing the variable from the value extracted from actual postgresql version command.

Reply:
I will see if I can extract it from `psql` that will save us in the future when we do upgrades as we would have little places to change. Or maybe even add it as an env variable in the main Dockerfile and we could then do install like ` apt-get install postgresql-$POSTGIS_VER-ogr-fdw